python/main.py

```diff
@@ -1,139 +1,176 @@
 import json
+import settings
+import ynab_client
 
-from flask import Flask, render_template, request, jsonify, redirect
+from flask import Flask, request, jsonify, redirect
 from datetime import datetime
 from dateutil.parser import parse
 from decimal import Decimal
 
 from pynYNAB.schema.budget import Transaction
 
-import settings
-import ynab_client
-
 app = Flask('Monzo to YNAB')
 app.config['DEBUG'] = settings.flask_debug
 
-if settings.sentry_dsn:
```

Comment:
Ah, totally! Sorry, I removed it because it wasn't in `.env.example` and thought it was not being used.